addons/core/functions/fnc_getConfigWeaponProperty.sqf

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getConfigWeaponProperty
+
+    Author(s):
+        Dorbedo
+
+    Description:
+    Gets a config property (getNumber/getText/getArray)
+    Only works for CfgWeapon.
+
+    Parameters:
+    0: STRING - Item classname
+    1: STRING - property
+    2: ANYTHING - Default (Optional)
+
+    Returns:
+    NUMBER or TEXT or ARRAY - Result
+
+    Example:
+        [_SrRadio, "tf_prototype", 0] call TFAR_fnc_getConfigWeaponProperty;
+*/
+
+params ["_item", "_property", ["_default", ""]];
+
+private _cacheName = (_item + _property);
+private _cachedEntry = TFAR_ConfigWeaponCacheNamespace getVariable _cacheName;
```

Comment:
Namespace could be GVAR. Don't really needs to though. But It's a Internal only thing so it might aswell.